models/turbine_models/custom_models/sd_inference/utils.py

```diff
@@ -36,13 +36,13 @@
     ],
     "unet": [
         "--iree-flow-enable-aggressive-fusion",
-        "--iree-flow-enable-fuse-horizontal-contractions=true",
+        "--iree-global-opt-enable-fuse-horizontal-contractions=true",
```

Comment:
Is change needed for current IREE?

Reply:
It seems to be, at least as of the 7/24 nightly. We might be using a newer version in the CI.

Reply:
7/24 is very old. Should test with latest or at least what iree-turbine pulls (which of right now is iree-compiler==20240828.999). Looks like the CI failures you are seeing are also due to flag changes with newer iree versions